sdk/storage/azblob/blockblob/client.go

```diff
@@ -165,6 +165,21 @@ func (bb *Client) Upload(ctx context.Context, body io.ReadSeekCloser, options *U
 	return resp, err
 }
 
+// PutBlobFromURL - The Put Blob from URL operation creates a new Block Blob where the contents of the blob are read from
+// a given URL. Partial updates are not supported with Put Blob from URL; the content of an existing blob is overwritten
+// with the content of the new blob. To perform partial updates to a block blob’s contents using a source URL, use the Put
+// Block from URL API in conjunction with Put Block List.
+// For more information, see https://learn.microsoft.com/rest/api/storageservices/put-blob-from-url
+func (bb *Client) PutBlobFromURL(ctx context.Context, copySource string, options *PutBlobFromURLOptions) (PutBlobFromURLResponse, error) {
+	opts, httpHeaders, leaseAccessConditions, cpkInfo, cpkSourceInfo, modifiedAccessConditions, sourceModifiedConditions := options.format()
```

Comment:
Is this our standard practice in azblob to return so many values from format method ()? Returning a structure will look much simpler in terms of code.

Reply:
Yes, this is the current practice seen throughout azblob. Changing the current practice would be inconsistent.